torch/ao/quantization/pt2e/utils.py

```diff
@@ -381,10 +380,9 @@ def _get_aten_graph_module_for_pattern(
             kwargs,
         ).module()
     else:
-        aten_pattern = capture_pre_autograd_graph(
-            pattern,  # type: ignore[arg-type]
-            example_inputs,
-            kwargs,
+        raise RuntimeError(
+            "capture_pre_autograd_graph is deprecated and will be deleted soon."
+            "Please use torch.export.export_for_training instead."
```

Comment:
This is a BC-breaking change right? Should we mark it as such in the PR title and description?

Reply:
> This is a BC-breaking change right? Should we mark it as such in the PR title and description?

sure!